app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventPresenter.java

```diff
@@ -38,17 +37,18 @@ public CreateEventPresenter(IEventRepository eventRepository) {
 
     @Override
     public void start() {
+        getView().attachCountryList(CurrencyUtils.getCountryList());
         getView().attachCurrencyCodesList(CurrencyUtils.getCurrencyCodesList());
 
-        //set default timezone
         List<String> timeZoneList = getView().getTimeZoneList();
         getView().setDefaultTimeZone(
             timeZoneList.indexOf(TimeZone.getDefault().getID())
         );
 
-        List<String> currencyList = CurrencyUtils.getCurrencyCodesList();
-        getView().setDefaultCurrency(
-            currencyList.indexOf(Currency.getInstance(Locale.getDefault()).getCurrencyCode())
+        List<String> countryList = CurrencyUtils.getCountryList();
```

Comment:
Cache country list rather than calling again and again

Reply:
It lives in `Presenter#start` method, which is called only once. How is this being used again and again?

Reply:
It's used twice in `start`